torch/testing/_internal/common_fsdp.py

```diff
@@ -976,18 +972,22 @@ def check_1d_sharded_parity(
         cls.assertEqual(replicated_name, clean_sharded_name)
         cls.assertIsInstance(sharded_param, DTensor)
         assert isinstance(sharded_param, DTensor)  # mypy
-        param_chunks = torch.chunk(replicated_param, world_size, dim=0)
-        cls.assertEqual(sharded_param._local_tensor, param_chunks[rank])
-        if not check_grads:
-            continue
+        mesh, placements = sharded_param.device_mesh, sharded_param.placements
+        if tuple(placements) == (Shard(0), Shard(0)):
```

Comment:
We will support this in the future with strided sharding. For now, FSDP's `(Shard(0), Shard(0))` differs from `distribute_tensor()`'s `(Shard(0), Shard(0))` since FSDP's one shards dim-1 (TP) first then dim-0, whereas `distribute_tensor()` shards dim-0 then dim-1.